core/src/main/scala/org/apache/spark/deploy/master/Master.scala

```diff
@@ -753,6 +757,7 @@ private[deploy] class Master(
     workers += worker
     idToWorker(worker.id) = worker
     addressToWorker(workerAddress) = worker
+    if (reverseProxy) webUi.addProxyTargets(worker.id, worker.webUiAddress)
```

Comment:
style: please use the "long form" of if statements (unless they're used as a ternary operator)